scripts/menu.gsc

```diff
@@ -536,10 +536,39 @@ MenuOptionsPlayer(menu, player)
             self addMenu(menu, "Upgraded Specials");
             self addOpt("Zitros Orbital Arbalest", &GiveClientWeapon, "launcher_standard_t8_upgraded", self);
             self addOpt("Thekrauss Refibrillator++", &GiveClientWeapon, "special_ballisticknife_t8_dw_upgraded", self);
-            if(BO4GetMap() == "Blood" || BO4GetMap() == "AO" || BO4GetMap() == "Tag" || BO4GetMap() == "Classified"){
-                self addOpt("Porters X2 Ray Gun", &GiveClientWeapon, "ray_gun_upgraded", self);
-                self addOpt("Porters Mark II Ray Gun", &GiveClientWeapon, "ray_gun_mk2_upgraded", self);
+            self addOpt("Porters X2 Ray Gun", &GiveClientWeapon, "ray_gun_upgraded", self);
+            self addOpt("Porters Mark II Ray Gun", &GiveClientWeapon, "ray_gun_mk2_upgraded", self);
+            if(BO4GetMap() == "Voyage"){
+                self addOpt("Decayed Kraken", &GiveClientWeapon, "ww_tricannon_earth_t8_upgraded", self);
+                self addOpt("Plasmatic Kraken", &GiveClientWeapon, "ww_tricannon_fire_t8_upgraded", self);
+                self addOpt("Purified Kraken", &GiveClientWeapon, "ww_tricannon_water_t8_upgraded", self);
+                self addOpt("Radiant Kraken", &GiveClientWeapon, "ww_tricannon_air_t8_upgraded", self);
+                self addOpt("Upgraded Kraken", &GiveClientWeapon, "ww_tricannon_t8_upgraded", self);
+            }
+            else if(BO4GetMap() == "Blood"){
+                self addOpt("Magmagat", &GiveClientWeapon, "ww_blundergat_fire_t8_upgraded", self);
+                self addOpt("AcidGat", &GiveClientWeapon, "ww_blundergat_acid_t8_upgraded", self);
+                self addOpt("Blundergat", &GiveClientWeapon, "ww_blundergat_t8_upgraded", self);
+            }
+            else if(BO4GetMap() == "AE"){
+                self addOpt("Hand of Ouranous", &GiveClientWeapon, "ww_hand_o_upgraded", self);
+                self addOpt("Hand of Hemera", &GiveClientWeapon, "ww_hand_h_upgraded", self);
+                self addOpt("Hand of Gaia", &GiveClientWeapon, "ww_hand_g_upgraded", self);
+                self addOpt("Hand of Charon", &GiveClientWeapon, "ww_hand_c_upgraded", self);
             }
+            else if(BO4GetMap() == "Tag"){
+                self addOpt("ThunderGun", &GiveClientWeapon, "thundergun_upgraded", self);
+                self addOpt("test", &GiveClientWeapon, "ww_tesla_gun_t8_upgraded", self);
```

Comment:
Did this work? If so, you should get the PAP name, and use it for the option.

Reply:
Also done